applications/user-generated-content.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Machine translation for user-generated content
+---
+
+# User-Generated Content
+
+Machine translation for **user-generated content (UGC)** is used mainly for **gisting**.
```

Comment:
I think the abbreviation should go outside the bold.

Reply:
In the new version, I didn't use the acronym at all. Do you think that I should keep it in that paragraph?